### ISSUE TITLE
copy does not work for tables which names starts with numbers

### ISSUE THREAD

**Issue by KES777** [citusdata/citus#545]
```
COPY "195_transaction" FROM '/tmp/tmp.csv' WITH CSV
WARNING:  Bad result from citus-work2:9701
DETAIL:  Remote message: syntax error at or near "_102016"
CONTEXT:  COPY 195_transaction, line 1: ""1","95382995","1000","INVOICE 95382995","","","","","60"
```

at the worker log:
```
ERROR:  syntax error at or near "_102016" at character 30
STATEMENT:  COPY public."195_transaction"_102016 FROM STDIN WITH (FORMAT BINARY)
```

How to reproduce: 
create table with numbers at the beginning and try to `copy` in it

**Comment by metdos** [comment]
Related with #82.

**Comment by KES777** [comment]
Should I close this issue or will you do this byself later?

**Comment by metdos** [comment]
Hey @KES777, first of all, thanks for trying out Citus and letting us know what we are missing. 

For this issue, I think we can keep it open, so whoever fixes the #82 can make sure that this problem is also fixed.
